src/shared/components/person/settings.tsx

```diff
@@ -45,11 +51,17 @@ import { SortSelect } from "../common/sort-select";
 import Tabs from "../common/tabs";
 import { CommunityLink } from "../community/community-link";
 import { PersonListing } from "./person-listing";
+import { InitialFetchRequest } from "shared/interfaces";
+
+type SettingsData = RouteDataResponse<{
```

Comment:
Your call, but seems weird to mix type and interface, and probably better to always use interface wherever possible.

Reply:
An interface won't work here. Interfaces can be used to defined the shape of a type, but they cannot be used to alias types like is being done here.